src/common/XAuth.cpp

```diff
@@ -29,6 +29,9 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <X11/Xauth.h>
+#ifdef __FreeBSD__
+#define HOST_NAME_MAX _POSIX_HOST_NAME_MAX
```

Comment:
Is `_POSIX_HOST_NAME_MAX` a FreeBSD-ism?

Reply:
The comment in `/usr/include/sys/syslimits.h` says:

> /*
 * We leave the following values undefined to force applications to either
 * assume conservative values or call sysconf() to get the current value.
 *
 * HOST_NAME_MAX
 *
 * (We should do this for most of the values currently defined here,
 * but many programs are not prepared to deal with this yet.)
 */

Reply:
I'm wondering whether `#ifndef HOST_NAME_MAX` is more generic in this case. Though nothing other than Linux and FreeBSD is currently supported anyway.